diff --git a/node_modules/@prisma/engines/dist/index.d.ts b/node_modules/@prisma/engines/dist/index.d.ts
index 2ade630..030e87a 100644
--- a/node_modules/@prisma/engines/dist/index.d.ts
+++ b/node_modules/@prisma/engines/dist/index.d.ts
@@ -1,4 +1,4 @@
-import { BinaryType as BinaryType_2 } from '@prisma/fetch-engine';
+import { BinaryType } from '@prisma/fetch-engine';
 import { enginesVersion } from '@prisma/engines-version';
 
 export declare const DEFAULT_CLI_QUERY_ENGINE_BINARY_TYPE = BinaryType.QueryEngineLibrary;
@@ -11,7 +11,7 @@ export declare function ensureBinariesExist(): Promise<void>;
  * Checks if the env override `PRISMA_CLI_QUERY_ENGINE_TYPE` is set to `library` or `binary`
  * Otherwise returns the default
  */
-export declare function getCliQueryEngineBinaryType(): BinaryType_2;
+export declare function getCliQueryEngineBinaryType(): BinaryType;
 
 export declare function getEnginesPath(): string;
 
